.changes/next-release/bugfix-ApacheHTTPClient-bf0095f.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "Apache HTTP Client",
```

Comment:
nit: this is not just applicable to Apache HTTP client right? Should we use `AWS SDK for Java v2` or whatever the right category for core?